src/packages/cli/src/__tests__/__snapshots__/integrate.postgresql.test.ts.snap

```diff
@@ -1,763 +1,781 @@
-exports['findOne - check typeof array for Json field with array_datamodel'] = `
```

Comment:
I only checked a few cases but it seems that the new results are faithful to the old.

Reply:
:+1: